packages/open-ui/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react';
+import { useAnimate } from 'react-simple-animate';
+import ReactDOM from 'react-dom';
+import classNames from 'classnames';
+import ScrollLock from 'react-scrolllock';
+
+import { Title } from 'components/Typography';
+import { Button } from 'components/Button';
+import { IntentOption } from 'interfaces/IntentOption';
+import { toSentenceCase } from 'utils/strings/to-sentence-case';
+import { CloseButton } from './CloseButton';
+import styles from './Modal.module.scss';
+
+interface ButtonProps {
+  onClick: () => void;
+  text: string;
+}
+
+interface ModalProps {
+  active: boolean;
+  close: () => void;
```

Comment:
@remotealex maybe propName as onModalClose --> for explicitely clearing what it is doing.

Reply:
`onModalClose` sounds like a callback to me. We could change it to `closeModal` if you want to be verbose?

Reply:
@remotealex We are currently passing a callback here to be called when modal is closed ryt ? I am sure I am missing something here bro